pkg/skaffold/update/update.go

```diff
@@ -43,23 +42,27 @@ func IsUpdateCheckEnabled() bool {
 	return v == "" || strings.ToLower(v) == "true"
 }
 
-// GetLatestVersion uses a VERSION file stored on GCS to determine the latest released version of skaffold
-func GetLatestVersion(ctx context.Context) (semver.Version, error) {
+// VersionCheck uses a VERSION file stored on GCS to determine the latest released version of skaffold
```

Comment:
please clarify the meaning of the returned values in the godoc

Reply:
Very good points. Your comments are quite related, which made me think that the API for `GetLatestVersion` is actually pretty bad. I think I have found a much better solution which addresses both your concerns.

Please let me know if you prefer those commits to be squashed.